### PR TITLE
#698 - edit schema

### PR DESCRIPTION
**Pull request by RChandler234**
## Changes

Edited the prisma schema to incorporate the risk log

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/docs/ContributorGuide.md) and reach out to your squad if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors
- [x] No newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (if applicable)
- [x] Remove any not-applicable sections
- [x] Assign the PR to yourself
- [x] PR is linked to the ticket
- [x] No `package-lock.json` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack

Closes # (issue #)

### CODE REVIEW

**Comment by jamescd18** [comment]
This looks good to me!

**Review by jamescd18 — APPROVED** [review]

**Review by jamescd18 — CHANGES_REQUESTED** [review]
just realized there’s like 4 things missing. I realized that because Risks aren’t gonna be facilitated through the change log, we wanna record more info:

- who created the risk
- who resolved the risk
- when the risk was resolved
- who deleted the risk

**Comment by RChandler234** [comment]
Ended up scrapping the uuid investigation in this PR, because I couldn't get the uuid's to auto-generate locally, and I wanna get this ticket done to unblock others; I'll open another ticket under this epic to look into the change before we deploy the feature and start getting risk data (probably wanna talk to @jamescd18 when we both have time because not sure how best to test that this works as we expect)

**Review by jamescd18 — APPROVED** [review]

**Review by anthonybernardi — DISMISSED** [review]
nice

File: src/backend/prisma/seed.ts [hunk 1]
```diff
@@ -17,6 +18,10 @@ const performSeed: () => Promise<void> = async () => {
     await prisma.user.create({ data: { ...seedUser } });
   }
 
+  for (const seedRisk of dbSeedAllRisks) {
+    await prisma.risk.create({ data: { ...seedRisk } });
```
Comment (jamescd18) [inline src/backend/prisma/seed.ts:22]: Pretty sure you're gonna need to do a connect to link it up with the user creating it. See change requests for an example probably

Comment (jamescd18) [reply src/backend/prisma/seed.ts:22]: Also might make more sense to have this come after projects

Comment (jamescd18) [reply src/backend/prisma/seed.ts:22]: And probably connect to the project

**Review by jamescd18 — CHANGES_REQUESTED** [review]

**Review by jamescd18 — APPROVED** [review]

**Pull request merged**